wled00/FX.cpp

```diff
@@ -675,7 +681,7 @@ uint16_t mode_twinkle(void) {
     uint32_t it = strip.now / cycleTime;
     
     // Initialize on first call
-    if (SEGENV.call == 0) 
+    if ((SEGENV.call == 0) || (SEGENV.aux1 > 1))
     {
```

Comment:
The suggested change was to add a value to the non-existent `aux2`. This method assumes that the `aux1` value was set by `hw_random` in the original code on line 750 to be some value (hopefully greater than 1).

Even if the data is not initialized, once the cycle completes, it gets initialized and works fine.